### PR TITLE
[edn/en] Add a short tutorial for EDN

### PR DESCRIPTION
**Pull request by jsyeo**

### CODE REVIEW

File: edn.html.markdown [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+```Clojure
+; Comments start with a semicolon.
+; Anythng after the semicolon is ignored.
+
+;;;;;;;;;;;;;;;;;;;
+;;; Basic Types ;;;
+;;;;;;;;;;;;;;;;;;;
+
+nil         ; or aka null
```
Comment (geoffliu) [inline edn.html.markdown]: "or" and "aka" mean the same thing here, keep only one of them.

File: edn.html.markdown [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+false
+
+; Strings are enclosed in double quotes
+"hungarian breakfast"
+"farmer's cheesy omelette"
+
+; Characters are preceeded by backslashes
+\g \r \a \c \e
+
+; Keywords starts with a colon. They behave like enums. Kinda
```
Comment (geoffliu) [inline edn.html.markdown]: Keywords start with

File: edn.html.markdown [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+; like symbols in ruby.
```
Comment (geoffliu) [inline edn.html.markdown]: Capital R

File: edn.html.markdown [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+:eggs
+:cheese
+:olives
+
+; Symbols are used to represent identifiers. You can namespace symbols by
```
Comment (geoffliu) [inline edn.html.markdown]: Do namespaces start with `#`s? If so please explain.

File: edn.html.markdown [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+#spoon
+#kitchen/spoon ; not the same as #spoon
+#kitchen/fork
+#github/fork   ; you can't eat with this
+
+; Integers and floats
+42
+3.14159
+
+; Lists are a sequence of values
```
Comment (geoffliu) [inline edn.html.markdown]: are sequences of

**Comment by jsyeo** [comment]
@geoffliu fixed! Thanks for reviewing! :sweat_smile:

File: edn.html.markdown [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+false
+
+; Strings are enclosed in double quotes
+"hungarian breakfast"
+"farmer's cheesy omelette"
+
+; Characters are preceeded by backslashes
+\g \r \a \c \e
+
+; Keywords start with a colon. They behave like enums. Kinda
```
Comment (geoffliu) [inline edn.html.markdown]: "Kinda" is still here.

**Comment by jsyeo** [comment]
Opps, I missed that. It's fixed now. @geoffliu

**Comment by geoffliu** [comment]
Cool, thanks!

**Pull request merged**